src/run-cache.test.ts

```
@@ -1275,4 +1275,71 @@ describe("RunCache", () => {
+      
+      // Advance timers to verify intervals were cleared
+      jest.advanceTimersByTime(2000);
+      await Promise.resolve(); // Flush microtasks
+      
+      // Listeners should not be called
+      expect(expiryListener).not.toHaveBeenCalled();
+      expect(refetchListener).not.toHaveBeenCalled();
+      
+      // Check that configuration is reset
+      expect(RunCache.getConfig()).toEqual({
+        maxSize: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)